src/output_queue/output_queue.py

```diff
@@ -50,7 +50,7 @@ def run(self):
             self._check_queue()
 
     # Tells the output thread that it should stop
-    def signal_stop(self):
+    def deactivate(self):
```

Comment:
Tests aren't passing anymore because you didn't rename this in the unit tests. (Though also it never seems to finish which is exactly what pytest-timeout is *supposed* to avoid...)

Reply:
ahhhhhhhhhhhhhhhhhhhhhhhhhhh

Reply:
fixed, approve plz